lib/percy/capybara/client.rb

```diff
@@ -17,14 +18,17 @@ class WebMockBlockingConnectionsError < Error; end
 
       attr_accessor :sprockets_environment
       attr_accessor :sprockets_options
-      attr_accessor :custom_loader
+      attr_accessor :loader
+      attr_accessor :loader_options
 
       def initialize(options = {})
         @failed = false
 
         @client = options[:client] || Percy.client
         @enabled = options[:enabled]
 
+        @loader_options = {}
```

Comment:
You could set this like `@loader_options = options[:loader_options] || {}` so people could set default loader options if initializing a custom Percy::Capybara::Client object (which is supported). IMO the only public interface should be `use_loader`, we should treat `initialize_loader` and the `capybara_client.loader_options` accessor as private (and ideally maybe we make them _actually_ private and an attr_reader).